app/models/case/sar/offender.rb

```diff
@@ -459,6 +461,10 @@ def use_subject_as_requester
     self.name = subject_full_name
   end
 
+  def set_case_originally_rejected
+    self.case_originally_rejected = true
```

Comment:
I assumed it would be ok to not give a value as false if its not a rejected case. So either nil or true.